### PR TITLE
Update phppsinfo.php with litespeed support

### PR DESCRIPTION
**Pull request by Vudubond**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update phppsinfo.php with litespeed support
| Type?             |  improvement
| BC breaks?        |  no
| Deprecations?     |  no
| Fixed ticket?     | Upload script on Litespeed server shows Apache Module (low performance)
| How to test?      | Upload script on Litespeed server shows Apache Module (low performance)
| Possible impacts? | Upload script on Litespeed server shows Apache Module (low performance)

### CODE REVIEW

**Review by 0x346e3730 — CHANGES_REQUESTED** [review]
Could you please fill out your PR description correctly ? Thank you !

**Comment by Vudubond** [comment]
Hello,

PR edited. Please check.

**Review by kpodemski — DISMISSED** [review]
Sorry for the late reply @Vudubond. I think there are some issues with code formatting. You should use 4 spaces. Maybe it could also say "Better performance", instead "Super performance".

File: phppsinfo.php [hunk 1]
```diff
@@ -164,14 +164,14 @@ public function checkAuth()
      */
     public function getVersions()
     {
-        $data = [
+       $data = [
```
Comment (kpodemski) [inline phppsinfo.php:167]:
```suggestion
        $data = [
```

File: phppsinfo.php [hunk 2]
```diff
@@ -164,14 +164,14 @@ public function checkAuth()
...
             'Web server' => [$this->getWebServer()],
             'PHP Type' => [
                 strpos(PHP_SAPI, 'cgi') !== false ?
                 'CGI with Apache Worker or another webserver' :
-                'Apache Module (low performance)'
-            ],
-        ];
+                (strpos(PHP_SAPI, 'litespeed') !== false ? '<span style="color: green">Litespeed (Super performance)</span>' : 'Apache Module (low performance)')
```
Comment (kpodemski) [inline phppsinfo.php:172]:
```suggestion
                (strpos(PHP_SAPI, 'litespeed') !== false ? '<span style="color: green">Litespeed (Better performance)</span>' : 'Apache Module (low performance)')
```

**Comment by Vudubond** [comment]
Requested Changes have been updated. Please check.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Tested and validated? Thank you @Vudubond and apologies for late feedback

**Pull request merged**